### PR TITLE
correct paths in install docs

### PR DESCRIPTION
**Pull request by stuarthicks**

### CODE REVIEW

**Comment by GeneticGenesis** [comment]
Good spot.

File: README.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ A Go library for generating MPEG-DASH manifests.
 This library uses [Glide](https://github.com/Masterminds/glide) to manage it's dependencies. Please refer to the Glide documentation to see how to install glide.
 
 ```bash
-mkdir -p $GOPATH/zencoder
-cd $GOPATH/zencoder
+mkdir -p $GOPATH/github.com/zencoder
```
Comment (brycefisher) [inline README.md]: So close!! Should be:

`$GOPATH/src/github.com/zencoder`

Comment (stuarthicks) [inline README.md]: What idiot review-passed this?! :stuck_out_tongue:

Comment (brycefisher) [inline README.md]: Honestly, I forget the `src` in the path EVERY time I try to use $GOPATH in my shell.... you'd think I'd remember when _I'm_ trying to use it.

**Pull request merged**